src/XenobladeChroniclesX/Mods/PhysicsSuperHighJump/rules.txt

```diff
@@ -1,8 +1,8 @@
 [Definition]
 titleIds = 0005000010116100,00050000101C4C00,00050000101C4D00
-name = "Jump higher"
-path = "Xenoblade Chronicles X/Mods/Physics/Jump to the Moon!"
-description = Increase horizontal and vertical velocity of jump (Vanilla values are 1.0 for both).
+name = "Super High Jump!"
+path = "Xenoblade Chronicles X/Mods/Physics/Super High Jump!"
```

Comment:
the (Vanilla values are 1.0 for both) isn't needed in the description because the x1 option for both distance and height have (default) in the name